src/pages/OnboardingEmployees/BaseOnboardingEmployees.tsx

```diff
@@ -63,13 +65,34 @@ function BaseOnboardingEmployees({shouldUseNativeStyles, route}: BaseOnboardingE
                     }
                     Welcome.setOnboardingCompanySize(selectedCompanySize);
 
+                    const {adminsChatReportID, policyID} = Policy.createWorkspace(undefined, true, '', Policy.generatePolicyID(), CONST.ONBOARDING_CHOICES.MANAGE_TEAM);
+
                     if (!onboardingPolicyID) {
-                        const {adminsChatReportID, policyID} = Policy.createWorkspace(undefined, true, '', Policy.generatePolicyID(), CONST.ONBOARDING_CHOICES.MANAGE_TEAM);
                         Welcome.setOnboardingAdminsChatReportID(adminsChatReportID);
                         Welcome.setOnboardingPolicyID(policyID);
                     }
 
-                    Navigation.navigate(ROUTES.ONBOARDING_ACCOUNTING.getRoute(route.params?.backTo));
+                    if (!NativeModules.HybridAppModule || selectedCompanySize === CONST.ONBOARDING_COMPANY_SIZE.MICRO) {
```

Comment:
A comment above this line to mirror line 80 would be good here. Mostly to explain what we do by navigating to `ONBOARDING_ACCOUNTING` and that this should only apply to `ONBOARDING_COMPANY_SIZE` (10 and less)